### ISSUE TITLE
resolve updates

### ISSUE THREAD

**Issue by substack** [defunctzombie/node-browser-resolve#56]
I just fixed some very tricky issues in resolve that were cropping up in browserify and are now part of the test suite. I recommend upgrading to 1.1.3, since the current tests fail when browser-resolve uses the old version.

The updates address some errors with the previous `opts.pathFilter` implementation and some very subtle bugs when there is a `./file.js` and a `./file` directory.

**Comment by substack** [comment]
Upgrading appears to breaks one of the browser-resolve tests. I'll iterate on resolve to see if I can get everything to pass.

**Comment by substack** [comment]
All fixed in 1.1.4. I also noticed in the current browser-resolve implementation that a package.json of:

``` json
{
  "browser": {
    "xyz": "wow.js"
  }
}
```

does not work for `require('foo/xyz')` but this does:

``` json
{
  "browser": {
    "./xyz": "wow.js"
  }
}
```

This is because internally a `./` is prepended in the pathFilter. This is something that can be handled later in a patch, but everything is ready to go right now.

**Comment by substack** [comment]
Just published under 1.7.1. The only change was upgrading resolve to 1.1.4.
